pkg/kubectl/cmd/util/factory_test.go

```diff
@@ -37,6 +37,7 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/sets"
```

Comment:
I think `Allow edits from maintainers.` means maintainers can edit your PR title and body. I'm not sure if they can edit your code...